internal/services/springcloud/spring_cloud_container_deployment_resource.go

```diff
@@ -115,30 +115,16 @@ func resourceSpringCloudContainerDeployment() *pluginsdk.Resource {
 							Type:     pluginsdk.TypeString,
 							Optional: true,
 							Computed: true,
-							ValidateFunc: validation.StringInSlice([]string{
-								"500m",
-								"1",
-								"2",
-								"3",
-								"4",
-							}, false),
+							// There are additional options when the service team allow them
```

Comment:
```suggestion
							// NOTE: we're intentionally not validating this field since additional values are possible when enabled by the service team
```